app.js

```diff
@@ -83,10 +84,14 @@ app.post("/api/shorten",
         .optional()
         .isString().withMessage("should be a string")
         .trim(),
+    body("expires")
+        .optional()
```

Comment:
Somewhat unsure as to what I actually think is best here you can see in #14, but I think it might be whorth considering making this mandatory

Reply:
I think you should either have to specify an expiration date or explicitly fill in some box saying it should last forever

Reply:
You don't want some links to expire, for example https://dsekt.se/github-access. However, as you mentioned in #14, it could perhaps be useful to optionally tie links to a mandate

Reply:
Yeah, if we want to allow permanent links I definitely think mandate-tied links will be needed, otherwise you will have to ask dsys or the original creator when they are to be updated some years later